### PR TITLE
Incident on wrong user getting access to Register - update request-an-account.html

### PR DESCRIPTION
**Pull request by MaeveRoseveare**
1. Added in section about 'Who can get an account'

For the first bullet - would a school who does placements ever think that because they offer placements they are an ITT provider?

2. edited bullet to say 'training provider you work for' instead of 'you represent'

### CODE REVIEW

File: app/views/request-an-account.html [hunk 1]
```diff
@@ -5,15 +5,22 @@
...
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <h1 class="govuk-heading-l govuk-!-margin-bottom-9">{{pageHeading}}</h1>
 
-    <p class="govuk-body">Follow these steps to get an account to use Register trainee teachers (Register).</p>
+    <h2 class="govuk-heading-m">Who can get a Register account</h2>
+    <p class="govuk-body">You can only get a Register account if you are:</p>
+    <ul class="govuk-list govuk-list--bullet">
+  <li>an initial teacher training (ITT) accredited provider</li>
+  <li>a lead school involved in School Direct training routes</li>
+  </ul>
```
Comment (edwardhorsford) [inline app/views/request-an-account.html]: Indentation needs fixing on these

Comment (edwardhorsford) [reply app/views/request-an-account.html]: Would it be worth linking to anything for each of these?

File: app/views/request-an-account.html [hunk 2]
```diff
@@ -5,15 +5,22 @@
...
 
 {% block content %}
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <h1 class="govuk-heading-l govuk-!-margin-bottom-9">{{pageHeading}}</h1>
 
-    <p class="govuk-body">Follow these steps to get an account to use Register trainee teachers (Register).</p>
+    <h2 class="govuk-heading-m">Who can get a Register account</h2>
+    <p class="govuk-body">You can only get a Register account if you are:</p>
```
Comment (edwardhorsford) [inline app/views/request-an-account.html]: could this imply you need to be both?

File: app/views/request-an-account.html [hunk 3]
```diff
@@ -26,7 +33,7 @@ <h2 class="govuk-heading-m">Step 2: Email us to request a Register account</h2>
     <p class="govuk-body">Once you have a DfE Sign-in account with a named email address, email <a href="mailto:becomingateacher@digital.education.gov.uk?subject=Request a Register account" class="govuk-link">becomingateacher@digital.education.gov.uk</a></p>
     <p class="govuk-body">In your email, tell us:</p>
     <ul class="govuk-list govuk-list--bullet">
-      <li>the name of the training provider or lead school you represent</li>
+      <li>the name of the training provider or lead school you work for</li>
```
Comment (edwardhorsford) [inline app/views/request-an-account.html]: Should this refer to accredited provider?

**Review by edwardhorsford — CHANGES_REQUESTED** [review]

File: app/views/request-an-account.html (hunk 1 above)
Comment (MaeveRoseveare) [reply app/views/request-an-account.html]: For accredited provider we could link to https://www.gov.uk/guidance/initial-teacher-training-itt-accreditation or https://www.gov.uk/government/collections/initial-teacher-training
I feel like the first link might be better as it explains there's a process to be an accredited provider, it's not just anyone

For lead school possibly: https://www.gov.uk/guidance/school-direct-guidance-for-lead-schools

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**